<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video Cutter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>VideoCutter.ico</normaloff>VideoCutter.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Vidéo sortie</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="videoInPushButton">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Paramètres</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="5">
        <widget class="QSpinBox" name="endSecSpinBox">
         <property name="toolTip">
          <string>Seconde de fin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QSpinBox" name="startMinSpinBox">
         <property name="toolTip">
          <string>Minute de début</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="endHourSpinBox">
         <property name="toolTip">
          <string>Heure de fin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QSpinBox" name="startSecSpinBox">
         <property name="toolTip">
          <string>Seconde de début</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Fin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="startHourSpinBox">
         <property name="toolTip">
          <string>Heure de début</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="endMinSpinBox">
         <property name="toolTip">
          <string>Minute de fin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Début</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>sec</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>h</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>min</string>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>sec</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Vidéo entrée</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="videoOutPushButton">
      <property name="text">
       <string>Parcourir</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="videoInLineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="cutvideoPushButton">
      <property name="text">
       <string>Couper !</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="videoOutLineEdit"/>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QProgressBar" name="progressBar">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="stopPushButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionLog"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>videoInLineEdit</tabstop>
  <tabstop>videoInPushButton</tabstop>
  <tabstop>videoOutLineEdit</tabstop>
  <tabstop>videoOutPushButton</tabstop>
  <tabstop>startHourSpinBox</tabstop>
  <tabstop>startMinSpinBox</tabstop>
  <tabstop>startSecSpinBox</tabstop>
  <tabstop>endHourSpinBox</tabstop>
  <tabstop>endMinSpinBox</tabstop>
  <tabstop>endSecSpinBox</tabstop>
  <tabstop>cutvideoPushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>videoInPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVideoInPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>755</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoOutPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onVideoOutPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>765</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cutvideoPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onCutVideoPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>701</x>
     <y>459</y>
    </hint>
    <hint type="destinationlabel">
     <x>594</x>
     <y>497</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopPushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>onStopCutPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onVideoInPushButtonClicked()</slot>
  <slot>onVideoOutPushButtonClicked()</slot>
  <slot>onCutVideoPushButtonClicked()</slot>
  <slot>onStopCutPushButtonClicked()</slot>
 </slots>
</ui>
